src/main/src/jetty/geoserver_data/wps/gogoduck.xml

```diff
@@ -2,4 +2,19 @@
     <fileLimit>10</fileLimit>
     <threadCount>1</threadCount>
     <urlSubstitution key="^">http://data.aodn.org.au/</urlSubstitution>
-</gogoduck>
\ No newline at end of file
+    <ncksPath>/usr/bin/ncks</ncksPath>
+    <ncpdqPath>/usr/bin/ncpdq</ncpdqPath>
+    <ncrcatPath>/usr/bin/ncrcat</ncrcatPath>
+    <timeField>time</timeField>
+    <fileUrlField>file_url</fileUrlField>
+    <unpack>true</unpack>
+    <globalAttributes>
+        <attribute key="title">title</attribute>
```

Comment:
no idea what an attribute key is - name is more readable for those who need to update this file.